App.js

```diff
@@ -235,6 +237,16 @@ export default class App extends React.Component {
     return i18n.t(scope, {locale: this.state.locale, ...options})
   }
 
+  changeAppType = () => {
+
+
+    this.setState({appType: this.state?.appType === 'store' ? 'retail' : 'store'})
```

Comment:
client API response now has a clientType field (FOOD_BEVERAGE, RETAIL). Please use this field to determine client type.